pkg/gofr/datasource/file/s3/fs.go

```diff
@@ -122,7 +120,7 @@ func (f *FileSystem) Connect() {
 // This method creates an empty file at the specified path in the S3 bucket. It first checks if the parent directory exists;
 // if the parent directory does not exist, it returns an error. After creating the file, it retrieves the file metadata
 // and returns a `file` object representing the newly created file.
```

Comment:
It no longer returns a file object.

You should talk about type assertion for getting result.

But for me it's strange. Returning an interface of with a few method could be an option, why removing the type and use any?

Reply:
We did it to complete remove the GoFr's dependency from the s3 datasource. Since the `file.File` interface was GoFr's interface defined in file package we made this changes to cut that dependency